<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWindow</class>
 <widget class="QDialog" name="LobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lobby window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu{
background-color:   rgb(50, 50, 50);
color: white;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
}
QMenu::item:disabled{
background-color: rgb(147, 147, 147);
color: rgb(90, 90, 90);
}

QMenu::item:disabled:selected{
background-color:  rgb(147, 147, 147);
}



QMenu::item:selected  {
background-color: rgb(198, 66, 0);
}

QMenu::separator {
height: 0px;
}


#LobbyWindow{
background-color:   rgb(50, 50, 50);
}

#ChatOpenButton{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/ChatIcon.svg);
}


#ChatOpenButton:hover{
icon:url(:/Lobby/Icons/ChatIcon_Hover.svg);
}

#ListUserOpenButtom{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/ListUser.svg);
}

#ListUserOpenButtom:hover{
icon:url(:/Lobby/Icons/ListUser_Hover.svg);
}

#BackWidgetButt{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/BackPageLobby.svg);
}

#BackWidgetButt:hover{
icon:url(:/Lobby/Icons/BackPageLobby_Hover.svg);
}

#NextWidgetButt{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/NextPageLobby.svg);
}

#NextWidgetButt:hover{
icon:url(:/Lobby/Icons/NextPageLobby_Hover.svg);
}

#AnswerButtom{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/MessageSendIcon.svg);
}

#AnswerButtom:hover{
icon:url(:/Lobby/Icons/MessageSendIcon_Hover.svg);
}

#SendMessageButton{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/MessageSendIcon.svg);
}

#SendMessageButton:hover{
icon:url(:/Lobby/Icons/MessageSendIcon_Hover.svg);
}

#InstallPackButtom{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/InstallPackIcon.svg);
}

#InstallPackButtom:hover{
icon:url(:/Lobby/Icons/InstallPackIcon_Hover.svg);
}

#StartStopTest{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/StartTest.svg);
}

#StartStopTest:hover{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/StartTest_Hover.svg);
}

#textBrowser{
border:none;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
}
QLabel{
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="LeftHeaderBody" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="ListUserOpenButtom">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="CentralHeaderBody" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7"/>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="RightHeaderBody" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="ChatOpenButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QFrame" name="ChatMenuFrame">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="BodyChatWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QWidget" name="ButtonChatWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="MessageLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SendMessageButton">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignBottom">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="InstallPackButtom">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="MainBody" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="CenterBody" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QStackedWidget" name="stackedWidget"/>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="rightMargin">
             <number>40</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Back</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="BackWidgetButt">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QPushButton" name="StartStopTest">
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Next</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="NextWidgetButt">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>22</width>
                   <height>22</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QWidget" name="BottomBody" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QLineEdit" name="AnswerLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignBottom">
          <widget class="QPushButton" name="AnswerButtom">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="ListUserFrame">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#LeftPageList{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/BackPageLobby.svg);
}

#LeftPageList:hover{
icon:url(:/Lobby/Icons/BackPageLobby_Hover.svg);
}

#RightPageList{
background-color:   rgb(50, 50, 50);
border: none;
icon:url(:/Lobby/Icons/NextPageLobby.svg);
}

#RightPageList:hover{
icon:url(:/Lobby/Icons/NextPageLobby_Hover.svg);
}





</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="HeaderListFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="NameList">
           <property name="text">
            <string>List users</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="BodyFrameListFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item alignment="Qt::AlignTop">
             <widget class="QStackedWidget" name="ListStack">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="ListPage">
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <layout class="QVBoxLayout" name="ListPageLayout"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="RatingPage">
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <widget class="QLabel" name="RatingLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPushButton" name="LeftPageList">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="RightPageList">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
